reckoner/cli.py

```diff
@@ -53,8 +50,14 @@ def cli(ctx, log_level, *args, **kwargs):
                                                                        'development.')
 def plot(ctx, file=None, dry_run=False, debug=False, only=None, helm_args=None, local_development=False):
     """ Install charts with given arguments as listed in yaml file argument """
-    h = Reckoner(file=file, dryrun=dry_run, debug=debug, helm_args=helm_args, local_development=local_development)
-    h.install(only)
+    try:
+        h = Reckoner(file=file, dryrun=dry_run, debug=debug, helm_args=helm_args, local_development=local_development)
+        # Convert tuple to list
+        only = list(only)
```

Comment:
Is an explicit conversion here the right thing to do or should the class handle poor input gracefuly by raising and error?

Reply:
Could go either way - this is why I prefer stronger typed languages :) -- python3 will get us type hinting at least - so something to look forward to.

Reply:
Do you have a preference? I can change if you do - It really doesn't matter to me either way